### PR TITLE
Update helpers.md

### PR DESCRIPTION
**Pull request by vimalgorasiya**
This PR improves grammar and conciseness in the `helper` module description for AdonisJS utilities.

### CODE REVIEW

**Comment by Julien-R44** [comment]
thanks but it seems less clear like this imo. not sure how it improves grammar and conciseness

**Comment by vimalgorasiya** [comment]
@Julien-R44 

Thank you for your feedback. I aimed to address the repetition of the word **the** in the original sentence. However, if you find the previous version clearer, I respect that perspective. Thank you.

**Comment by Julien-R44** [comment]
Okay i am not a native English speaker, so maybe I'm wrong. @RomainLanz any thoughts ?

File: content/docs/reference/helpers.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Helpers reference
 
-AdonisJS bundles the utilities the framework uses into the `helpers` module and makes them available to your application code. Since these utilities are already installed and used by the framework, the `helpers` module does not add any additional bloat to your `node_modules`.
+AdonisJS bundles the utilities into the `helpers` module and makes them available to your application code. Since these utilities are already installed and used by the framework, the `helpers` module does not add any additional bloat to your `node_modules`.
```
[the remark this answers is not in the document]
Comment (RomainLanz) [reply content/docs/reference/helpers.md:3]: It sounds better like that and keeps the same intent.

**Pull request merged**